core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -22,9 +22,11 @@ import java.lang.management.ManagementFactory
 import java.net._
 import java.nio.ByteBuffer
 import java.util.{Properties, Locale, Random, UUID}
-import java.util.concurrent.{ThreadFactory, ConcurrentHashMap, Executors, ThreadPoolExecutor}
+import java.util.concurrent._
 import javax.net.ssl.HttpsURLConnection
 
+import org.apache.spark.network.util.JavaUtils
```

Comment:
Tiny, but this import is not in the right place. If we're doing another pass, might be worth a fix.